extra/FindNanopb.cmake

```diff
@@ -360,6 +391,10 @@ foreach(FIL ${_nanopb_hdrs})
   list(APPEND NANOPB_HDRS "${${FIL}__nano_pb_file}")
 endforeach()
 
+# Create the library target
+add_library(nanopb STATIC ${NANOPB_SRCS})
```

Comment:
But shouldn't this be only created if used in 'target mode'?

Reply:
Also suggest to set EXCLUDE_FROM_ALL

Reply:
> But shouldn't this be only created if used in 'target mode'?

I think there should be a globally available target `nanopb` immediately after calling `find_package(Nanopb)`, so that's why it always gets created.  If someone wants to do manual encoding/decoding (something like `protoscope`), then they could link directly to `nanopb` instead of a target generated by `nanopb_generate_cpp()`.